spec/extractors/reek_extractor_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+describe RepoAnalyzer::ReekExtractor, repo_analyzer_extractor_helpers: true do
+  describe "#extract" do
+    let(:reek_result_content) do
+      [
+        {
+          "context" => "ApplicationCable::Channel",
+          "lines" => [2],
+          "message" => "has no descriptive comment",
+          "smell_type" => "IrresponsibleModule",
+          "source" => "spec/assets/test_project/app/channels/application_cable/channel.rb",
+          "documentation_link" => "http://gh.com/reek/blob/v6.1.4/docs/Irresponsible-Module.md"
+        },
+        {
+          "context" => "ApplicationCable::Connection",
+          "lines" => [2],
+          "message" => "has no descriptive comment",
+          "smell_type" => "IrresponsibleModule",
+          "source" => "spec/assets/test_project/app/channels/application_cable/connection.rb",
+          "documentation_link" => "http://gh.com/reek/blob/v6.1.4/docs/Irresponsible-Module.md"
+        },
+        {
+          "context" => "AbastibleClient#parse_prices_by_type",
+          "lines" => [189],
+          "message" => "has the variable name 'e'",
+          "smell_type" => "UncommunicativeVariableName",
+          "source" => "spec/assets/test_project/app/clients/abastible_client.rb",
+          "name" => "e",
+          "documentation_link" => "http://gh.com/reek/blob/v6.1.4/docs/Uncommunicative-Variable-Name.md"
+        }
+      ].to_json
+    end
+
+    before do
+      allow(OutputUtils).to receive(:with_captured_stdout).and_return(reek_result_content)
+    end
+
+    let(:expected) do
+      {
+        "warnings" =>
+          {
+            "IrresponsibleModule" =>
+              [
+                {
+                  "lines" => [2],
+                  "message" => "has no descriptive comment",
+                  "source" => "app/channels/application_cable/channel.rb",
+                  "name" => nil,
+                  "documentation_link" => "http://gh.com/reek/blob/v6.1.4/docs/Irresponsible-Module.md" },
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/MultilineHashBraceLayout: Closing hash brace must be on the line after the last hash element when opening brace is on a separate line from the first hash element.